src/content/peerconnection/codec_constraints/css/main.css

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
```

Comment:
s/2015/2017

Reply:
Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.